test/python/algorithms/test_vqe.py

```diff
@@ -147,7 +147,7 @@ def test_basic_aer_qasm(self):
 
         # TODO benchmark this later.
         result = vqe.compute_minimum_eigenvalue(operator=self.h2_op)
-        self.assertAlmostEqual(result.eigenvalue.real, self.h2_energy, places=3)
+        self.assertAlmostEqual(result.eigenvalue.real, -1.86823, places=2)
```

Comment:
Why does it not converge to `self.h2_energy`? Do you need to change the options of SPSA?

Reply:
I am not sure... But I found it depends on the random seed and the number of measurements (1024 now).

Reply:
As I wrote the Pull Request message, this was original in opflow.

Reply:
Yes, it does depend on seeds and shots. If it is what it was before a recent change then it seems fine. As you can see its actually found a value below the true value due to the sampling noise.